docker-compose.yaml

```diff
@@ -20,9 +26,13 @@ services:
   # its own port, and have its own entry in the workspace.yaml file that's loaded by dagit.
   user_code:
     env_file:
-      - .env.staging
+      - .env
     container_name: user_code
-    image: code-server #mathewpugh/fons-code-server:latest
+    expose:
+      - "4000"
+    ports:
+      - "4000:4000"
+    image: sfdl/fons-code-server-org:latest   #sfdl/fons-code-server-client:latest
```

Comment:
What does this comment indicate?

Reply:
I could probably have two docker compose files or maybe it would be better to have two code servers setup.

For the data platform, there are two code servers that the service needs to work through. Data is ingested into the LA setup (fons-code-server-client), and then handed over to the code server for the organisation (fons-code-server-org).  I have it commented here so I can easily switch between them. 

There may be better ways to do this though.

Reply:
Ah is the switching for local testing then?

Reply:
Yes, this compose file isn't used other than testing at the moment.